chain/chain/src/metrics.rs

```diff
@@ -239,3 +239,29 @@ pub(crate) static APPLY_CHUNK_RESULTS_CACHE_MISSES: LazyLock<IntCounterVec> = La
     )
     .unwrap()
 });
+
+pub(crate) static STATE_TRANSITION_DATA_GC_TOTAL_ENTRIES: LazyLock<IntGauge> =
+    LazyLock::new(|| {
+        try_create_int_gauge(
+            "near_state_transition_data_gc_total_entries",
+            "Number of entries in state transaction data store column",
+        )
+        .unwrap()
+    });
+
+pub(crate) static STATE_TRANSITION_DATA_GC_CLEARED_ENTRIES: LazyLock<IntCounter> =
+    LazyLock::new(|| {
+        try_create_int_counter(
+            "near_state_transition_data_gc_cleared_entries",
+            "Time taken to do garbage collection of state transaction data",
+        )
+        .unwrap()
+    });
+
+pub(crate) static STATE_TRANSITION_DATA_GC_TIME: LazyLock<Histogram> = LazyLock::new(|| {
+    try_create_histogram(
```

Comment:
please specify buckets explicitly here, we want to have values tailored for our use case

Reply:
Added in [afcb6e8](https://github.com/near/nearcore/pull/13175/commits/afcb6e8a285a0d6443a263c69f3f3f8274114a8c).

I assumed you want here more fine-grained buckets than defaults, let me know if that's not fine-grained enough.